src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import classNames from 'classnames';
+import { SortType } from '../../types/SortType';
+
+type Props = {
+  activeTodosLength: number;
+  sortType: SortType;
+  setSortType: (sortBy: SortType) => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  setSortType,
+  sortType,
+  activeTodosLength,
+}) => {
+  const handleSortType = (value: SortType) => {
+    setSortType(value);
+  };
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        { activeTodosLength }
+        items left
+      </span>
+
+      {/* Active filter should have a 'selected' class */}
+      <nav className="filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: sortType === SortType.ALL,
+          })}
+          onClick={() => handleSortType(SortType.ALL)}
```

Comment:
```suggestion
          onClick={() => setSortType(SortType.ALL)}
```
`handleSortType` wrapper is redundant